mantidproject/mantid
# Rename Indexes in IDA

**Description of work.**

This PR renames the Indexes in IDA to be more clear in what they are. Primarily it reanames TableDatasetIndex to WorkspaceID indicating that the index was used to locate a workspace within a list of workspaces, as opposed to WorkspaceIndex which locates a spectra within a Workspace.

**To test:**

This should not change anty functionality. so long as it builds and tests pass it whould be fine. But open the interface and make sure there are no crashes when running.

Fixes #29207

*This does not require release notes* because **it is not a change to functionality.**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
